ast/src/parsed/mod.rs

```diff
@@ -1150,6 +1215,28 @@ impl<E> Children<E> for IfExpression<E> {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct StructExpression<E = Expression<NamespacedPolynomialReference>> {
+    pub name: SymbolPath,
```

Comment:
Oh wait, I think this needs to be NamespacedPolynomialReference or something, because we can have stuff like `super::mod::StructName::<int, int[]>`.